tutorials/hist/hsumTimer.C

```diff
@@ -12,14 +13,14 @@
 ///
 /// \author Rene Brun
 
-Float_t ratio;
+Float_t ratioratio;
```

Comment:
Python raises an exception when running the notebook, but I suppose it is an underlying compilation error. The traceback is as follows:
```
input_line_32:2:10: error: redefinition of 'ratio' as different kind of symbol
 Float_t ratio;
         ^
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/ratio:234:29: note: previous definition is here
class _LIBCPP_TYPE_VIS_ONLY ratio
                            ^
```